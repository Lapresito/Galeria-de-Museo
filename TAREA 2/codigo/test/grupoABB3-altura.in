crearGrupoABB
alturaGrupoABB
liberarGrupoABB
crearGrupoABB
crearVisitante 10 Artemio Perez 21
agregarAGrupoABB
alturaGrupoABB
liberarGrupoABB
crearGrupoABB
crearVisitante 5 Casimira Fernandez 52
agregarAGrupoABB
crearVisitante 3 Ubaldo Fernandez 19
agregarAGrupoABB
alturaGrupoABB
crearVisitante 2 Esmeralda Lopez 21
agregarAGrupoABB
alturaGrupoABB
crearVisitante 7 Zenobia Sanchez 33
agregarAGrupoABB
alturaGrupoABB
crearVisitante 6 Filemon Rodriguez 22
agregarAGrupoABB
alturaGrupoABB
crearVisitante 8 Brunilda Ruiz 31
agregarAGrupoABB
alturaGrupoABB
liberarGrupoABB
Fin
