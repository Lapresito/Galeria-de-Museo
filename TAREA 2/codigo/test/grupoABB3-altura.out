Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> El grupoABB ha sido creado de forma exitosa.
2> La altura del grupo es 0.
3> Grupo liberado con exito.
4> El grupoABB ha sido creado de forma exitosa.
5> Visitante creadox de forma exitosa.
6> Se ha agregado el visitante al grupo de forma exitosa.
7> La altura del grupo es 1.
8> Grupo liberado con exito.
9> El grupoABB ha sido creado de forma exitosa.
10> Visitante creadox de forma exitosa.
11> Se ha agregado el visitante al grupo de forma exitosa.
12> Visitante creadox de forma exitosa.
13> Se ha agregado el visitante al grupo de forma exitosa.
14> La altura del grupo es 2.
15> Visitante creadox de forma exitosa.
16> Se ha agregado el visitante al grupo de forma exitosa.
17> La altura del grupo es 3.
18> Visitante creadox de forma exitosa.
19> Se ha agregado el visitante al grupo de forma exitosa.
20> La altura del grupo es 3.
21> Visitante creadox de forma exitosa.
22> Se ha agregado el visitante al grupo de forma exitosa.
23> La altura del grupo es 3.
24> Visitante creadox de forma exitosa.
25> Se ha agregado el visitante al grupo de forma exitosa.
26> La altura del grupo es 3.
27> Grupo liberado con exito.
28> Fin.
